includes/discovery/os/nos.inc.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
 if (!$os) {
-    if (strstr($sysObjectId, '.1.3.6.1.4.1.1588.2.2.1.1.1.5') || stristr($sysDescr, 'Brocade Communications Systems')) {
-        $os = 'nos';
-    }
-}
+  if (strstr($sysDescr, "Brocade VDX")) { $os = "nos"; }
```

Comment:
As far as I could find in the MIB documentations, that oid doesn't even exist anymore. checking for Brocade VDX as sysDescr is the correct way and will catch nos devices :)